script.embuary.helper/resources/lib/helper.py

```diff
@@ -44,24 +45,6 @@
 
 ########################
 
-def get_kodiversion():
-    build = xbmc.getInfoLabel('System.BuildVersion')
-    return int(build[:2])
-
-
-def addon_setting(skin,setting,save=False):
-    profile = xbmc.getInfoLabel('System.ProfileName')
-    setting_id = skin + '_' + profile + '_' + setting
-    skin_version = xbmcaddon.Addon(skin).getAddonInfo('version')
-
-    if not save:
-        if ADDON.getSetting(id=setting_id) == skin_version:
-            return True
-        return False
-    else:
-        ADDON.setSetting(id=setting_id, value=skin_version)
-
-
```

Comment:
Default log level should be debug

Reply:
it only logs with logging enabled in the addon settings, but I'll change that to always show the logging with enabled debug log

Reply:
I think it look OK now. @enen92 if you agree we can merge.